qadence/mitigations/readout.py

```diff
@@ -17,6 +18,37 @@ def corrected_probas(p_corr: npt.NDArray, T: npt.NDArray, p_raw: npt.NDArray) ->
     return norm(T @ p_corr.T - p_raw.T, ord=2) ** 2
 
 
+def mle_solve(p_raw: npt.NDArray) -> npt.NDArray:
+    """
+    Computes the MLE probability vector in accordance to the.
```

Comment:
Was the . added automatically here?

Reply:
This is the docstring formatter in the pre-commits [here](https://github.com/pasqal-io/qadence/blob/fcd34dad8905420c08af9417d857680daa4f2454/.pre-commit-config.yaml#L28). It is sometimes a bit dumb and brutal in its formatting.